# This file was autogenerated by uv via the following command:
#    uv pip compile --universal -o requirements-dev.txt requirements-dev.in --python-version 3.9
-e file:.
    # via -r requirements-dev.in
astor==0.8.1
    # via flake8-qgis
cfgv==3.4.0
    # via pre-commit
colorama==0.4.6 ; sys_platform == 'win32'
    # via pytest
coverage==7.6.1
    # via pytest-cov
debugpy==1.8.6
    # via -r requirements-dev.in
distlib==0.3.8
    # via virtualenv
exceptiongroup==1.2.2 ; python_full_version < '3.11'
    # via pytest
filelock==3.16.1
    # via virtualenv
flake8==7.1.1
    # via
    #   -r requirements-dev.in
    #   flake8-qgis
flake8-qgis==1.0.0
    # via -r requirements-dev.in
identify==2.6.1
    # via pre-commit
iniconfig==2.0.0
    # via pytest
mccabe==0.7.0
    # via flake8
mypy==1.11.2
    # via -r requirements-dev.in
mypy-extensions==1.0.0
    # via mypy
nodeenv==1.9.1
    # via pre-commit
packaging==24.1
    # via pytest
platformdirs==4.3.6
    # via virtualenv
pluggy==1.5.0
    # via
    #   pytest
    #   pytest-qt
pre-commit==3.8.0
    # via -r requirements-dev.in
pycodestyle==2.12.1
    # via flake8
pyflakes==3.2.0
    # via flake8
pyqt5-stubs==5.15.6.0
    # via -r requirements-dev.in
pytest==8.3.3
    # via
    #   -r requirements-dev.in
    #   pytest-cov
    #   pytest-qgis
    #   pytest-qt
pytest-cov==5.0.0
    # via -r requirements-dev.in
pytest-qgis==2.1.0
    # via -r requirements-dev.in
pytest-qt==4.4.0
    # via -r requirements-dev.in
pyyaml==6.0.2
    # via pre-commit
ruff==0.6.7
    # via -r requirements-dev.in
tomli==2.0.1 ; python_full_version <= '3.11'
    # via
    #   coverage
    #   mypy
    #   pytest
typing-extensions==4.12.2
    # via mypy
virtualenv==20.26.5
    # via pre-commit
